### DISCUSSION TITLE
French translation - addition

### DISCUSSION THREAD

**Discussion by Michel74960** [mozilla/policy-templates#850] [Q&A]
Hi everyone,
I have translated in French some policies which weren't and made some corrections/modifications to others.
How can I bring to you the ADML file?
I think it would be necessary to verify them, so what is the proper format to view the modifications between the files?
Cordially,
Michel

**Comment by mkaply** [comment]
Do you have any experience with Git?

Typically you would fork the repo, make the change and submit a push request.

If you want, you can send me the file and I can create the pull request.

**Comment by Michel74960** [comment]
Hi Michael,
sorry I don't have any experience with GitHub!
You can find file attached, in .ods format (LibreOffice Calc):
[Liste ADMX 01-01-2022 traduction.ods](https://github.com/mozilla/policy-templates/files/7831521/Liste.ADMX.01-01-2022.traduction.ods)
The second column shows the translations done and other modifications.

My work is not complete, I have some questions later.
Cordially,
Michel

**Comment by mkaply** [comment]
Can you provide me the ADML instead of the doc file?
